examples/ios/Flower/Sources/Flower/FlowerProto/transport.proto

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Adap GmbH. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ==============================================================================
+
+syntax = "proto3";
+
+package flower.transport;
```

Comment:
`transport.proto` changes quite a bit in Flower 1.0, some messages have been renamed to follow a more consistent naming scheme and there is a new `Status` message included in all of the client messages. I'd propose to merge this PR with the version of `transport.proto` that is currently included (which should be in line with the version of `transport.proto` used in Flower 0.19) and then create a follow-up PR to make the necessary changes for Flower 1.0.